circle.yml

```diff
@@ -10,23 +10,26 @@ dependencies:
   override:
     - pip install -r requirements_test.txt
     - pip install -e .
-    - npm install
-    - npm install -g jshint
+    # This cannot be done because pip on Circle doesn't understand our sdist
+    # - make sdist
+    # - pip install dist/ka-lite-"$(python setup.py --version)".tar.gz
   post:
     - if [[ ! -e sc-latest-linux/bin/sc ]]; then wget https://saucelabs.com/downloads/sc-latest-linux.tar.gz && tar -xzf sc-latest-linux.tar.gz &&  mv sc-*-linux sc-latest-linux; fi
 
 test:
   override:
-    - node build.js
-    - kalite manage init_content_items
+    - make assets
     - kalite start --traceback -v2
+    - kalite start --settings=kalite.project.settings.disk_based_cache --traceback -v2
```

Comment:
Indeed - the line above is now preferred, as the disk_based_cache as was is removed in develop.